Makefile

```
@@ -299,3 +300,8 @@ clobber_logs: ## Purges logs and tmp/
 
 tidy: clobber_assets clobber_logs ## Remove assets, logs, and unused gems, but leave DB alone
 	bundle clean
+
+idv_events: app/services/idv/engine/events.rb
+
+app/services/idv/engine/events.rb: app/services/idv/engine/events.yml scripts/build-idv-events.rb
+	scripts/build-idv-events.rb $< > $@.tmp && mv $@.tmp $@
```

what's the purpose of the `.tmp` file and not just writing? guarding on failures in the script?

I would expect:
```suggestion
	scripts/build-idv-events.rb $< > $@
```